drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -399,10 +399,17 @@ def form_definition(id:)
 
     field :external_form_definition, Types::Forms::FormDefinition, null: true do
       argument :identifier, String, required: true
+      argument :id, ID, required: false # not required for backwards compat
```

Comment:
Feels like this would be simpler as a new query `external_form_definition_by_id`, and we can deprecate and eventually remove the old query. This is a little funky as-is because the `identifier` is "required" even though it's not necessary when looking up by ID. Although, I wonder whether we need a special query for looking up by id, vs resolving the definition on the submission https://github.com/greenriver/hmis-frontend/pull/909/files#r1742449259